testing/test_doctest.py

```diff
@@ -14,9 +16,9 @@
 
 
 class TestDoctests:
-    def test_collect_testtextfile(self, testdir):
-        w = testdir.maketxtfile(whatever="")
-        checkfile = testdir.maketxtfile(
+    def test_collect_testtextfile(self, pytester):
```

Comment:
To get some "typing coverage" of `PyTester` is would nice to type-annotate these tests, e.g.

```py
def test_collect_testtextfile(self, pytester: PyTester) -> None:
```

here and below. Only if you feel like it...

Reply:
Great idea, will do! 👍